code/__HELPERS/global_lists.dm

```diff
@@ -50,7 +50,17 @@
 
 	GLOB.emote_list = init_emote_list()
 
-	init_subtypes(/datum/crafting_recipe, GLOB.crafting_recipes)
+	init_crafting_recipes(GLOB.crafting_recipes)
+
+/// Inits the crafting recipe list, sorting crafting recipe requirements in the process.
+/proc/init_crafting_recipes(list/crafting_recipes)
+	if(!istype(crafting_recipes))
+		crafting_recipes = list()
```

Comment:
```suggestion
	if(!islist(crafting_recipes))
		crafting_recipes = list(crafting_recipes)
```
Did you mean something like this?

Reply:
This was copied from init_subtypes on line 68. I guess I can customise it in this context. GLOB.crafting_recipes will always be an empty list at this point of code.